pkg/api/pod/util.go

```diff
@@ -454,6 +456,9 @@ func GetValidationOptionsFromPodSpecAndMeta(podSpec, oldPodSpec *api.PodSpec, po
 		// if old spec has OS field set, we must allow it
 		opts.AllowOSField = opts.AllowOSField || oldPodSpec.OS != nil
 
+		// if old spec has status.hostIPs downwardAPI set, we must allow it
+		opts.AllowHostIPsField = opts.AllowHostIPsField || hasUsedDownwardAPIFieldPathWithPodSpec(oldPodSpec, "status.hostIPs")
```

Comment:
does this apply here?
we are setting the pod.status.hostIPs field and this is about pod.spec ... the status is generated

Reply:
Enable this feature and create a pod that uses this feature, if this feature is disabled, must have tolerate this downward. pod using this function when disabled cannot be created.

Reply:
ic, now I understand, so, you are saying that if I have:

```
apiVersion: v1
kind: Pod
metadata:
  name: test-pod
spec:
  containers:
    - name: test-container
      image: gcr.io/google_containers/busybox
      command: [ "/bin/sh", "-c", "env" ]
      env:
        - name: MY_POD_HOST_IPS
          valueFrom:
            fieldRef:
              fieldPath: status.hostIPs
  restartPolicy: Never
```

that pod should be allowed despite the feature gate status?

I defer to @liggitt  and @thockin , they are the experts on this area

Reply:
Yes, that's the general semantic.